src/js/MainController.js

```diff
@@ -133,9 +133,12 @@
         $scope.time = year;
       });
 
-      $scope.deviceSwitcherHref = gaPermalink.getHref({ mobile: mobile });
+      // Create switch device url
+      var switchToMobile = '' + !(gaBrowserSniffer.mobile || gaBrowserSniffer.embed);
+      $scope.host = {url: $window.location.host}; // only embed 
```

Comment:
"And redirect to embed if in iframe? "

no

- since we will have applications as http://www.wildruhezonen.ch/wr400.php
http://www.bfe-gis.admin.ch/storymaps/EA_SGT/ etc whcih want to use the
responsive version .
- the use of the desktop app in an iframe is permitted


make sure we dont breal those apps above....

On Fri, Mar 6, 2015 at 10:04 AM, Gilbert Jeiziner <notifications@github.com>
wrote:

> In src/js/MainController.js
> <https://github.com/geoadmin/mf-geoadmin3/pull/2197#discussion_r25932565>:
>
> > @@ -133,9 +133,12 @@
> >          $scope.time = year;
> >        });
> >
> > -      $scope.deviceSwitcherHref = gaPermalink.getHref({ mobile: mobile });
> > +      // Create switch device url
> > +      var switchToMobile = '' + !(gaBrowserSniffer.mobile || gaBrowserSniffer.embed);
> > +      $scope.host = {url: $window.location.host}; // only embed
>
> What doest he comment mean here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geoadmin/mf-geoadmin3/pull/2197/files#r25932565>.
>